apps/app/src/app/pages/(components)/components/(autocomplete)/autocomplete--async.example.ts

```diff
@@ -0,0 +1,46 @@
+import { ChangeDetectionStrategy, Component, inject, resource, signal } from '@angular/core';
+import { FormBuilder, Validators } from '@angular/forms';
+import { HlmAutocomplete } from '@spartan-ng/helm/autocomplete';
+
+@Component({
+	selector: 'spartan-autocomplete-async',
+	imports: [HlmAutocomplete],
+	template: `
+		<hlm-autocomplete [options]="options.value()" (filterChange)="_filter.set($event)" />
```

Comment:
When we start searching it shows "No options found" - is there a way to showing a loading state to differentiate between no options because we are loading and no options because there were no matches?

Reply:
Good idea! I added an option for loading indicator which changes the content in the empty state. I added a spinner to the example. When you enter `empty` it should show "No options found" instead of the loading spinner.

What do you think of the approach?